__tests__/app.test.js

```diff
@@ -524,3 +524,23 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+describe("GET /api/users", () => {
+  test("returns a 200 status code", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+  test("returns an array of objects with correct properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
```

Comment:
if this is an array it's a bit of a vague test, but I would argue you don't need it anyway since it's proved once you iterate over it on line 539